source/_patterns/03-templates/04-header.mustache

```diff
@@ -2,13 +2,24 @@
 	<div class="row header-row">
 	{{! This is the Promo/Bonus banner that is shown in the XXlarge screen state}}
 		<div class="show-for-xxlarge-up xxlarge-4 xxlarge-centered columns promo-space">
-			{{#bonus}}
-				<div style="font-weight: normal;">Select a borrower to </br>
+			{{#lending-reward-session}}
+				<div style="font-weight: normal;">
 					<span style="font-weight: bold;">
-						<a class="dark_green_bg_link" href="{{bonus.href}}">lend your {{bonus.amount}} {{bonus.label}}</a>
+						<a class="dark_green_bg_link" href="{{lending-reward-session.href}}">Make a Kiva loan</a>
```

Comment:
fyi you can just have {{href}} here, since this block is scoped to #lending-reward-session. how it is here will also work, though.